webapp/pages/admin/categories.vue

```diff
@@ -31,7 +31,7 @@ export default {
   },
   apollo: {
     Category: {
-      query: gql(`
+      query: gql`
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-08-13T11:27:40Z" title="Tuesday, August 13th 2019, 1:27:40 pm +02:00">Aug 13, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -31,7 +31,7 @@ export default {
   },
   apollo: {
     Category: {
-      query: gql(`
+      query: gql`
```


:open_mouth: code creep!!!

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-08-14T06:15:53Z" title="Wednesday, August 14th 2019, 8:15:53 am +02:00">Aug 14, 2019</time>_

---


What means `code creep`?
I couldn’t find a understandable translation or explanation … @mattwr18

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-08-14T08:48:45Z" title="Wednesday, August 14th 2019, 10:48:45 am +02:00">Aug 14, 2019</time>_

---


Oh sorry, it's when changes are made that are not directly related to the scope of the issue/Pull Request